lib/jsdom/living/navigator/PluginArray-impl.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+const idlUtils = require("../generated/utils");
+
+exports.implementation = class PluginArray {
+  refresh() {}
+
+  get length() {
+    return 0;
+  }
+
+  item() {
+    return null;
+  }
+
+  namedItem() {
+    return null;
+  }
+
+  get [idlUtils.supportedPropertyIndices]() {
+    return new Set();
```

Comment:
I believe you could return an empty array (which might be slightly cheaper), but the difference is insignificant anyway.